README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
this code is elegant and awesome

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Elegant

Reply:
fix the logic
Core logic that is related to solving a business or domain problem should exist outside of a framework.

📄 [Principles.dev](https://principles.dev/p/)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
michael need to fix this-- the logic isn't
Conditional logic should avoid double negatives.

📄 [Principles.dev](https://principles.dev/p/)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
2 comment to see link

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix